### ISSUE TITLE
Missing support for assistive technologies (accessibility)

### ISSUE THREAD

**Issue by richardgrey** [signavio/react-mentions#451]
**Steps to reproduce:**
Use any assistive technology (VoiceOver on Mac will work) and try to trigger suggestions.

**Expected behaviour:**
Users of assistive technology should be able to identify a list of suggestions and to control it with keyboard or technologies (i.e. voice commands).

**Observed behaviour:**
No support for assistive technologies. Users can't identify the existence of a suggestions list or control it with assistive technologies.

**Workaround:**
To apply [Combobox pattern](https://www.w3.org/TR/wai-aria-practices-1.1/examples/combobox/aria1.1pattern/listbox-combo.html%23ex2_label) recommended by W3C WAI.

**Comment by richardgrey** [comment]
Hi @jfschwarz! Do you think this PR will make sense for the library?
